lib/licenses.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+const licenseTypes = {
+  // permissive licenses - not public domain and not viral
+  'permissive': {
+    spdxLicenseIds: ['AFL-3.0', 'Apache-2.0', 'Artistic-2.0', 'BSD-2-Clause', 'BSD-3-Clause', 'BSD-3-Clause-Clear',
+      'BSL-1.0', 'CC-BY-4.0', 'ECL-2.0', 'ISC', 'MIT', 'MS-PL', 'NCSA', 'PostgreSQL', 'Zlib'],
+    color: 'blue'
+  },
+  // viral licenses require 'Disclose source' (https://choosealicense.com/appendix/#disclose-source)
+  // or 'Same license' (https://choosealicense.com/appendix/#same-license)
+  'viral': {
```

Comment:
Though we don't print this anywhere, I'd suggest we use `copyleft`, which is a descriptive yet neutral term.

Reply:
done